File: mobile/app/src/main/java/com/ticketbus/mobile/TicketBusApp.java
package com.ticketbus.mobile;

import android.app.Application;

public class TicketBusApp extends Application {
    private static TicketBusApp instance;

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
    }

    public static TicketBusApp getInstance() {
        return instance;
    }
}

File: mobile/app/src/main/java/com/ticketbus/mobile/TicketBusApp.kt
package com.ticketbus.mobile

import android.app.Application
import androidx.work.*
import com.ticketbus.mobile.offline.SyncWorker
import java.util.concurrent.TimeUnit

class TicketBusApp : Application() {
    companion object { lateinit var instance: TicketBusApp }
    override fun onCreate() {
        super.onCreate()
        instance = this
    }
}
